### PR TITLE
Modified slider for vertical navigation on mobile

### PR DESCRIPTION
**Pull request by markschwindt**

### CODE REVIEW

File: src/components/TopNav.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default class TopNav extends Component {
   render () {
     const {isOpen} = this.state;
     return (
-      <Headroom disableInlineStyles calcHeightOnResize>
+      <Headroom disableInlineStyles calcHeightOnResize upTolerance={100}>
```
Comment (cmil) [inline src/components/TopNav.js:83]: I notice that in iPhone Safari there glitch when scrolling up. The top of the first card is hidden underneath the navbar and only after some delay fully reveals. Is this maybe headroom related?

dracor-capture

File: src/index.css [hunk 2]
```diff
@@ -1050,7 +1044,7 @@ table.table.corpus th:focus {
   .navbar-brand {
     width: 2.2em;
     height: 2.2em;
-    margin: 1em;
+    margin: 1rem;
```
[the remark this answers is not in the document]
Comment (markschwindt) [reply src/index.css]: Since `em` is a relative unit, margins around the logo got recalculated after reactivating headroom, making mobile navigation bigger than it should be. Falling back to `rem` root size provided a temporary solution. However, I've already simplified margins handling on mobile all together in the new commit.

File: src/slick-theme.css [hunk 3]
```diff
@@ -4,15 +4,6 @@
...
 }
-.slick-slider::before {
-    content: '';
-    height: 12.1rem;
-    background: var(--main);
-    display: block;
-    position: absolute;
-    top: -5em;
-    width: 100%;
-}
```
[the remark this answers is not in the document]
Comment (markschwindt) [reply src/slick-theme.css:15]: I've assumed that 'slick-theme.css' was a file for our customizations. There are no recommendations for whether it should or should not be edited in the developer's documentation. It does, however, make sense to use our own custom theme file for adjustments.

File: src/components/TopNav.js (hunk 1 above)
Comment (markschwindt) [reply src/components/TopNav.js:83]: There are indeed multiple small issues regarding height calculation by headroom. Thank you for noticing that! I'll see whether it can be solved in the future. Maybe we should even look for an alternative package.
